azurerm/resource_arm_policy_assignment.go

```diff
@@ -90,6 +119,15 @@ func resourceArmPolicyAssignmentCreate(d *schema.ResourceData, meta interface{})
 		assignment.AssignmentProperties.Description = utils.String(v)
 	}
 
+	if v := d.Get("identity_type").(string); v != "" {
+		assignment.Identity = &policy.Identity{}
+		assignment.Identity.Type = policy.ResourceIdentityType(*(utils.String(v)))
```

Comment:
we should be able to make this:

```suggestion
		assignment.Identity.Type = policy.ResourceIdentityType(v))
```

Reply:
also if this isn't set should we be submitting `none` to the API?

Reply:
👍